### DISCUSSION TITLE
License

### DISCUSSION THREAD

**Discussion by francksa** [yhoriuchi/projoint#39] [General]
What are the restrictions of use for projoint? more specifically interested in the statistical approach to random noise errors.

**Comment by yhoriuchi** [comment]
There is no restriction on using this package. But please make sure to cite our paper! https://gking.harvard.edu/conjointE

**Comment by francksa** [comment]
Thanks ! For those using commercial SaaS products to run their conjoints ,is there a way to call the specific module that handles errors or does it only work with your own configuration ?Thanks much! On Apr 13, 2024, at 2:14 PM, Yusaku Horiuchi ***@***.***> wrote:﻿
There is no restriction on using this package. But please make sure to cite our paper! https://gking.harvard.edu/conjointE

**Comment by yhoriuchi** [comment]
Hi. I am not fully sure what your question is. But we design all
package-specific error messages. If you happen to encounter any errors,
please don't hesitate to let us know.

**Comment by francksa** [comment]
thanks! I was just saying that many conjoint users will be using existing
commercial solutions by various companies (e.g. Sawtooth, conjointly,
qualtrics, etc)- and i was wondering if your proposed code could
potentially be used in those other solutions (one usually can 'edit' and
change a few things)  or if the code only worked in your package. I get
that the approach could be applied, but was wondering if the code itself
was 'transportable' and if it'd cause licensing issues in the event it
could be transported into other applications
best!

**Reply by yhoriuchi** [reply]
Oh, I see. I understood. As of now, we are not considering the possibility of our package to be used in other commercial solutions. We'll think about the licensing issues. Thank you!

**Reply by aaronrkaufman** [reply]
Hello!

I am also not sure I understand your question, but I will take a crack at it.

Our software is an R package for **analyzing** conjoint data collected in Qualtrics. Any conjoint data you collect that can be converted into a format similar to a Qualtrics output will be compatible with our software. If you have SaaS software that also analyzes conjoint data, it is not possible to use our tool; you'd need to download the conjoint data and analyze it in R.
